packages/core/src/attestation/Attestation.chain.ts

```diff
@@ -8,20 +8,17 @@ import { DecoderUtils } from '@kiltprotocol/utils'
 import type { AccountId, Hash } from '@polkadot/types/interfaces'
 import { ConfigService } from '@kiltprotocol/config'
 import { BlockchainApiConnection } from '@kiltprotocol/chain-helpers'
-import Identity from '../identity/Identity'
 import Attestation from './Attestation'
 import type { DelegationNodeId } from '../delegation/DelegationDecoder'
 
 const log = ConfigService.LoggingFactory.getLogger('Attestation')
 
 /**
  * @param attestation
- * @param identity
  * @internal
  */
 export async function store(
-  attestation: IAttestation,
-  identity: Identity
```

Comment:
soon this will require an identity again (a did then), but for now removing the parameter seems to be the only way